packages/mutation-testing-elements/src/typings/highlight.js.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'highlight.js/lib/highlight' {
+  interface HighlightResult {
+    relevance: number;
+    value: string;
+  }
+  function registerLanguage(name: string, language: Language): void;
+  function highlight(language: string, code: string): HighlightResult;
+  function highlightBlock(code: HTMLElement): void;
+}
+
+declare module 'highlight.js/lib/languages/javascript' {
+  export = Language;
+}
+declare module 'highlight.js/lib/languages/typescript' {
+  export = Language;
+}
+declare module 'highlight.js/lib/languages/scala' {
+  export = Language;
+}
+declare module 'highlight.js/lib/languages/java' {
```

Comment:
Why is Java imported? As there are no frameworks using this report that have Java support. I feel like we should either import all Highlight.js languages, or add them on a case by case basis (preferred)

Reply:
True. Removed 👍